### PR TITLE
Handling dark/light modes app-wide

### PR DESCRIPTION
**Pull request by matthewmain**
So I finally dove into the react for this one, which is why I'm requesting your review in case I did anything wrong or messed something up.

What's happening here is pretty much just handling dark/light mode outside the game screen. Now when you toggle dark/light mode, the logo toggles between the light and dark designs to match the background. But adding this meant that the logo/background stayed in the same light state when you navigated to any of the dashboard screens. So, to keep the dark background in the dashboard screens when light mode is turned on for the game screen, I overrode the light mode settings when any of the dashboard screens are rendered, then replaced the light mode settings when the "resume" button is clicked. 

***ESPECIALLY CHECK*** `menu.js` where in the `resumeGame` function I set `props.appState.showGame` to `true` ([here](https://github.com/matthewmain/kiss_the_sky/pull/106/files#diff-256dceae6197be64c2ec491859566d03R86)). I needed something to track whether the game screen was being displayed or not, and I _think_ this is what you'd intended to use this for. But if this is for something else and updating it here will throw something else off, just let me know and I'll try to create an independent variable instead.

dl

### CODE REVIEW

**Comment by robbobfrh84** [comment]
OH awesome! Welcome to react land! lol. I'll take a look.

File: webapp/client/src/components/Menu/menu.js [hunk 1]
```diff
@@ -64,10 +65,26 @@ class Landing extends Component {
...
+      document.body.style.background = "#FFFFFF"
+      document.getElementById("logo").src = Logo_light;
+    }
+  }
+
   resumeGame = () => {
     if (window.gamePaused && window.gameHasBegun) {
       window.resume()
       this.state.history.push('/game')
+      this.props.appState.set({ showGame: true })
```
Comment (robbobfrh84) [inline webapp/client/src/components/Menu/menu.js]: ... this is a weird react thing, and a weird method I built to handle the weird thing. but whenever changing any of the  states for any elements in the component, you actually have call a the setState method to change the 
 value throughout the component(s)... So like this > `this.setState({ some_var: "new value" })` But, that'll only change the state of the component that you're in. And, I often want to change site-wide value. so the appState.set() essentially relays to the highest parent component to change all the "ShowGame" references through the site....

... So, you pretty much dove into the most complicated aspect of this site.

**Comment by robbobfrh84** [comment]
So I just updated that one little issue , but it looks great! good job. You're a React Dev now.

**Comment by matthewmain** [comment]
Nice, good to know! Nothing like jumping into the most complicated part to start figuring shit out. Makes me want to learn more react. Thanks!

**Pull request merged**